src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,17 @@
+.catalog {
+
+  &__card {
+    width: $card-size;
+    border: 1px solid #f3f3f3;
+    display: flex;
+    flex-direction: column;
+    margin-right: $card-margin;
+    margin-bottom: 46px;
+    padding: 32px 16px 16px;
+    border-radius: 5px;
```

Comment:
only margins should be added to the element class name
other styles should be added to main BEM class (card)
because if you move for now your card out of catalog, it should has styles, that you've added to the element class